components/src/core/ListItemTag/ListItemTag.tsx

```diff
@@ -40,6 +45,7 @@ const ListItemTagRender: React.ForwardRefRenderFunction<unknown, ListItemTagProp
     const {
         classes: userClasses,
         label,
+        variant,
         // ignore unused vars so that we can do prop transferring to the root element
         /* eslint-disable @typescript-eslint/no-unused-vars */
         fontColor,
```

Comment:
These unused props don't need to be deconstructed here; I think you can delete `fontColor` and `backgroundColor`.

Reply:
We destructure them so that we can pass all of the `otherTypographyProps` to the Typography component. If we don't destructure them, then when we pass `otherTypographyProps` to the Typography component, it will complain that 'backgroundColor is not a known property of Typography'.

The comment here tries to explain that. We do this in a couple of our components. As of yet, I haven't thought of a better way of doing it.

Reply:
I just ran into this issue while doing some Drawer refactors; this makes sense now.  It's not intuitive though for first-timers; I think there's gotta be a better way to do this, but I cannot think of a solution either.    This is fine for now :D